### PR TITLE
Made some CR fields optional and set defaults for the optional fields

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Most fields are optional and the default value for template validator replicas is 2.

The minimal PR looks like:
```
apiVersion: ssp.kubevirt.io/v1beta1
kind: SSP
metadata:
  name: example-ssp
  namespace: kubevirt
spec:
  commonTemplates:
    namespace: kubevirt

```

**Release note**:
```release-note
NONE
```

This PR takes over code from: #37

### CODE REVIEW

File: internal/operands/node-labeller/defaults.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package node_labeller
+
+const (
+	kubevirtNodeLabellerDefaultImage = "quay.io/kubevirt/node-labeller:v0.2.0"
+	kvmInfoNfdDefaultImage           = "quay.io/kubevirt/kvm-info-nfd-plugin:v0.5.8"
+	kvmCpuNfdDefaultImage            = "quay.io/kubevirt/cpu-nfd-plugin:v0.1.1"
+	libvirtDefaultImage              = "docker.io/kubevirt/virt-launcher:v0.21.0"
```
Comment (ksimon1) [inline internal/operands/node-labeller/defaults.go:7]: I think we can update this image to version `v0.35.0`

Comment (akrejcir) [reply internal/operands/node-labeller/defaults.go:7]: Ok, I will open another PR for it.

**Pull request merged**